.changeset/chilled-pens-fetch.md

```diff
@@ -0,0 +1,5 @@
+---
+"@khanacademy/wonder-blocks-popover": patch
+---
+
+Fix button text in the Popover With initialFocusId Storybook story
```

Comment:
Could you please change this file to have an empty message?

The short answer is that Storybook/Test only changes shouldn't be generating a new version of the WB package as this is not affecting the bundle that is later downloaded in webapp. This allow us to avoid publishing changes that are not required in npm.

```suggestion
---
---
```

Reply:
Apologies for not mentioning this before. We are planning to revamp the WB contribution docs in Confluence sometime in January or February to make all of these clearer.